tools/c7n_index/c7n_index/metrics.py

```diff
@@ -549,5 +550,5 @@ def index_resources(
         cli()
     except Exception as e:
         import traceback, pdb, sys
-        print traceback.print_exc()
+        print(traceback.print_exc())
```

Comment:
can you remove this line and the additional import statement at the top (line 14)